roles/wisdom_llm_load_test/files/llm-load-test_pod.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: llm-load-test
+ namespace: wisdom # {{ wisdom_namespace }}
+spec:
+  containers:
+  - name: llm-load-test
+    image: quay.io/dagray/llm-load-test:latest
+    imagePullPolicy: Always
+    command: ["/bin/sh"]
+    args: ["-c", "cd /src/llm-load-test && python3 ghz_frontend.py --wisdom_experiment"]
```

Comment:
I prefer writing this like that, so that it looks like a usual script
```
        command:
        - bash
        - -c
        args:
        - |
          set -o errexit
          set -o pipefail
          set -o nounset
          set -o errtrace
          set -x
```